src/app/components/Search/search-utils.ts

```diff
@@ -121,27 +128,21 @@ export function isSearchValid(searchTerm: string) {
 export const getSearchTermFromRequest = (request: Request) =>
   new URL(request.url).searchParams.get('q')?.trim() ?? ''
 
-export const searchParamLoader = async ({ request, params }: LoaderFunctionArgs) => {
-  const { network } = params
-  if (!!network && !RouteUtils.getEnabledNetworks().includes(network as Network)) {
+export const useParamSearch = () => {
+  const network = useNetworkParam()
+
+  if (!!network && !RouteUtils.getEnabledNetworks().includes(network)) {
     throw new AppError(AppErrors.InvalidUrl)
   }
 
-  const searchTerm = getSearchTermFromRequest(request)
+  const searchTerm = useSearchParams()[0].get('q')?.trim() ?? ''
   const normalized = Object.fromEntries(
     Object.entries(validateAndNormalize).map(([key, fn]) => [key, fn(searchTerm)]),
   ) as { [Key in keyof typeof validateAndNormalize]: string | undefined }
   return {
     searchTerm,
     ...normalized,
-    // TODO: remove conversion when API supports querying by EVM address
-    // TODO: without async conversion, this won't need to even be a loader
-    evmBech32Account: normalized.evmAccount ? await getEvmBech32Address(normalized.evmAccount) : undefined,
```

Comment:
So EVM support querying was added to the API?

Reply:
I don't think so. I moved this code to its buddies at https://github.com/oasisprotocol/explorer/blob/14de628f5304bed3c53c5c193976072d38393195/src/app/components/Search/search-utils.ts#L105-L111

Reply:
Now that it isn't async